docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_IDE.xml

```diff
@@ -724,12 +724,10 @@
                   <entry><emphasis role="bold">ECL Watch Graph
                   Control</emphasis></entry>
 
-                  <entry>While the graph viewer control is built in to ECL
-                  IDE, ECL Watch requires this control to be installed so that
-                  it can display graphs properly. The currently installed
-                  version is identified and the version number of the new
-                  version shown. You should only adjust this setting if
-                  instructed to do so by customer support.</entry>
+                  <entry>The graph viewer control is built in to the ECL IDE.
```

Comment:
This option is no longer in the IDE's Preferences > Other  tab. You should remove the entire row